packages/calypso-build/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@automattic/calypso-build",
+	"version": "1.0.0",
+	"description": "Shared build configurations.",
```

Comment:
Nitpick, I'd remove the trailing period.

Reply:
Also, maybe include the word 'Calypso' -- e.g. `Shared Calypso build configuration files`?

Reply:
should we consider calling packages "Automattic" instead of "Calypso"?

Reply:
There were also proposals about calling it a "Muriel SDK" 🙂

Reply:
> should we consider calling packages "Automattic" instead of "Calypso"?

They will be in the `@automattic` npm org, so I'd rather not duplicate that name part (`@automattic/automattic-build` sounds a bit silly TBH)

> There were also proposals about calling it a "Muriel SDK"  I think that the "Calypso" name, at least for now, is the most fitting description of the package content.

Agree with Jarda here :+1:

Reply:
3a180b2